### PR TITLE
Polish translation

### LINKED ISSUE #42: Polish (Polski) <pl.cfg>
# Localization - ![Polski][PO] Polish (Polski) <pl.cfg>

## To Be Done by :octocat: contributor

* Some helpful reading
  * [Readme on localization][lclztn]
  * [Quickstart guide to localization][qs]
  * these should help guide you
  * both are in markdown {text}
  * if you are feeling very helpful, kindly translate these as well. ;P
* GameData (done by contributor)
  * [ ] add Localization \ <pl.cfg> 🗃️
* Translation
  * [ ] ![Polski][PO] Polish (Polski) <pl.cfg>

1. Fork this repo
2. Create a branch name it 'translations' or localizations (suggestions)
3. goto `GameData/.../Localization/`
4. copy <en-us.cfg>
5. rename to <pl.cfg>
6. open the file and translate everything to the right of `#??? = `
7. change the `en-us` to `pl`
8. please include an attribution to you in the localization file, something like:
  * `// Translated by: [GitHubUserName](https://github.com/GitHubUserName)`
  * alternatively if you want another name from the KSP forums...
9. when done, PR it and you should reference this issue
10. After it has been merged by me, you may delete your fork.

have fun and thank you! :octocat: 🎃

## GitHub

* [x] assign
  * [x] zer0Kerbal
  * [x] Repo Localization Project
  * [x] Label: type: localization
  * [x] Label: help-wanted
  * [x] Milestone: Localization - Master

### below done by @zer0Kerbal

* Update
  * Documentation
    * [ ] [readme.md] 🔢
    * [ ] [release.md] 🧾
  * docs/
    * [ ] docs/[localization.md]
  * Update Social Media
  * [x] GitHub (automated) (zer0Kerbal/zer0Kerbal)
  * [x] CurseForge
  * [x] Reddit
  * [x] Twitter @zer0Kerbal
  * [x] Forum
  * [x] Patreon
  * [x] SpaceDock
  * [x] CKAN (automated)

[lclztn]: https://github.com/zer0Kerbal/zer0Kerbal/blob/master/Localization/readme.md  "Localization"
[qs]: https://github.com/zer0Kerbal/zer0Kerbal/blob/master/Localization/quickstart.md "Quick Start"

[PO]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/PO.png "Polski"

this file: © All Rights Reserved by zer0Kerbal

### PR DESCRIPTION
**Pull request by Monniasza**
Closes #42

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -130,7 +128,7 @@ config, parts, resources
 
 ### Legal Mumbo Jumbo (License *provenance*)
 
-#### Author (2) - [`zer0Kerbal`][zer0Kerbal]
```
Comment (zer0Kerbal) [inline readme.md:133]: thank you. missed that.

File: readme.md [hunk 2]
```diff
@@ -169,10 +167,8 @@ Comment, click, like, share, up-vote, subscribe
 [discu]: https://github.com/zer0Kerbal/SimpleHyperdrive/discussions "Discussions"
 [forum]: https://forum.kerbalspaceprogram.com/index.php?/topic/212069-*/ "SimpleHyperdrive Forum Thread"
 [issue]: https://github.com/zer0Kerbal/SimpleHyperdrive/issues "Issues"
-[markt]: https://zer0kerbal.github.io/SimpleHyperdrive/Marketing "Marketing Slicks"
```
Comment (zer0Kerbal) [inline readme.md:172]: I will add the missing marketing doc.

File: readme.md [hunk 3]
```diff
@@ -169,10 +167,8 @@ Comment, click, like, share, up-vote, subscribe
...
 [notic]: https://zer0kerbal.github.io/SimpleHyperdrive/Notices "Notices"
 [pages]: https://zer0kerbal.github.io/SimpleHyperdrive "GitHub Pages"
-[parts]: https://zer0kerbal.github.io/SimpleHyperdrive/PartsCatalog "Parts Catalog"
```
Comment (zer0Kerbal) [inline readme.md:175]: this file / link is valid.

File: readme.md [hunk 4]
```diff
@@ -196,7 +192,7 @@ Comment, click, like, share, up-vote, subscribe
 
 [LIC:url]: https://www.gnu.org/licenses/gpl-2.0-standalone.html "GPL-2.0"
 [LIC:log]: https://i.postimg.cc/9FrwMgK6/GPL-17x17.png "GPL-2.0"
-[LIC:shd]: https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/zer0Kerbal/SimpleHyperdrive/master/json/license.json
```
Comment (zer0Kerbal) [inline readme.md:199]: again, thank you.

File: readme.md (hunk 3 above)
Comment (Monniasza) [reply readme.md:175]: The ingame title was misleading for the ION tank and hyperdrive stabilizer

Comment (Monniasza) [reply readme.md:175]: I've readded it.

**Review by zer0Kerbal — APPROVED** [review]

**Comment by zer0Kerbal** [comment]
Thank you @Monniasza !
new release out - hope you like the edition name!

**Pull request merged**